site/data/i18n/en.json

```diff
@@ -186,13 +186,13 @@
 	"menu-footer-about-military-text": "Public service and military",
 	"menu-footer-about-military-link": "https://www.canada.ca/en/government/publicservice.html",
 	"menu-footer-about-news-text": "News",
-	"menu-footer-about-news-link": "http://news.gc.ca/",
```

Comment:
I'll update it in a sec. Was working at the time I sent this PR. My change on this line is also causing GitHub's merge conflict warning.

A few days after I opened this PR, #1213 was opened and merged. It changed the link in question to point to http://www.servicecanada.gc.ca/gcnews. But since that's an unrelated domain for a site that was recently migrated into Canada.ca, IMO it'd be more ideal to make the link go directly to the news site.